### ISSUE TITLE
How to deploy TURN & STUN service?

### ISSUE THREAD

**Issue by msh-nayan** [ddssingsong/webrtc_android#9]
hello @ddssingsong 
i've cloned both android & server side code. Android project is imported on Android Studio properly. But how can I start the server? I've installed NodeJS on my system (Ubuntu 18.04 LTS). 

You mentioned this "Deploy turn and stun services : https://github.com/coturn/coturn.git", then start "server.js". 

BUT how can I deploy "turn" & "stun" services?

Kindly help.

**Comment by ddssingsong** [comment]
ok. just google "How to deploy coturn"

and first item is ...The first is what you want, or the second

https://www.microsoft.com/developerblog/2018/01/29/orchestrating-turn-servers-cloud-deployment/

**Comment by msh-nayan** [comment]
@ddssingsong 
thanks for your suggestion. rather than setting up "coturn", I've decided to use public STUN & TURN services. So, I've checked your this answer (https://github.com/ddssingsong/webrtc_android/issues/4#issuecomment-523309355). I've put this `iceServers` value on `SkyRTC-client.js` file inside `const iceServer` value. Should I do anything more ?

After this, I ran `node server.js ` command to start the server and on terminal I can see the following output 

> Server running at http://0.0.0.0:3000/

I clicked (or copy paste) the link to open it on browser (Firefox or Chrome) but nothing happened. I saw this reply of you https://github.com/ddssingsong/webrtc_android/issues/4#issuecomment-523264207, I clicked on this URL (https://118.25.25.147#123456789) and on browser I can see permission is needed for Camera & Microphone; then I can see my preview. But nothing happened on my output link after running the server using `node server.js`? 

What I'm missing? Kindly reply. 

Thanks :)
